dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReader.java

```diff
@@ -157,12 +157,12 @@ private PagingQueryProvider createQueryProvider(DataSource dataSource) throws Ex
         factoryBean.setFromClause(
                 "FROM " +
                         "b150_snpcontigloc loc JOIN " +
-                        "b150_contiginfo ctg ON ctg.ctg_id = loc.ctg_id JOIN " +
+                        "b150_contiginfo ctg ON ctg.contig_gi = loc.ctg_id JOIN " +
```

Comment:
Is this a typo?

Reply:
No, there are 2 columns with identical values in that table and, in a very counter-intuitive decision, the primary key is `contig_gi` and not `ctg_id`.